addon/components/power-select/options.ts

```diff
@@ -54,22 +59,24 @@ export default class Options extends Component<Args> {
       if (optionIndex === null) return;
       action(this._optionFromIndex(optionIndex), select, e);
     };
-    element.addEventListener('mouseup', (e) => findOptionAndPerform(this.args.select.actions.choose, this.args.select, e));
+    this.mouseUpHandler = (e: MouseEvent): void => findOptionAndPerform(this.args.select.actions.choose, this.args.select, e);
+    element.addEventListener('mouseup', this.mouseUpHandler);
     if (this.args.highlightOnHover) {
-      element.addEventListener('mouseover', (e) => findOptionAndPerform(this.args.select.actions.highlight, this.args.select, e));
+      this.mouseOverHandler = (e: MouseEvent): void => findOptionAndPerform(this.args.select.actions.highlight, this.args.select, e);
+      element.addEventListener('mouseover', this.mouseOverHandler);
     }
     if (this.isTouchDevice) {
-      let touchMoveHandler = () => {
+      this.touchMoveHandler = (_: TouchEvent): void => {
         this.hasMoved = true;
         if (element) {
-          element.removeEventListener('touchmove', touchMoveHandler);
+          element.removeEventListener('touchmove', this.touchMoveHandler);
```

Comment:
Just one suggestion. Can you declare these event handlers that are not used in templates as _private_?

Reply:
Oh, right. Nevermind then.